### ISSUE TITLE
Adding modules does not work in Safari

### ISSUE THREAD

**Issue by valtism** [okikio/bundlejs#23]
Just found out about this project. Looks awesome, especially because Bundlephobia has been pretty flaky as of late.

In Safari, when selecting a package from the search select box, clicking add does nothing. Looks like the search results box is closing before the selection is made. No console output unfortunately.

https://user-images.githubusercontent.com/1286001/170160117-e1b4c98c-454d-4dea-9030-02b735006d88.mov

**Comment by okikio** [comment]
I'm having trouble recreating this issue (I don't have a mac so that might be why 😅). Right now I'm using [Gnomes Epiphany browser](https://wiki.gnome.org/Apps/Web) on [WSL](https://docs.microsoft.com/en-us/windows/wsl/install) to test [WebKit](https://webkit.org/), but as I don't have a Mac I can't test on Safari directly. Could you please clear the cache and try clicking the add button again.

**Comment by valtism** [comment]
Yeah, cleared cache and is still persisting. I would say that Safari is handling focus states differently so focus is either being dropped or is not considered within the input so :focused-within turn off before the click goes through. 

I think it would be better to handle the open state as actual state in a `useSignal` because `:focused-within` does feel a little hacky. 

I can have a look at fixing this myself later if you are struggling with replication, but I may take a while to get around to it.

**Comment by okikio** [comment]
I'll try to get on it as soon as possible, I am currently working on porting the entire project over to Astro

**Comment by valtism** [comment]
Yeah no rush. Still works with manually typing in the package name. Will be keeping an eye on this project because it looks super nice 👍

**Comment by MauriceArikoglu** [comment]
Can confirm it **NOT** working on any iPhone browser.

**Comment by okikio** [comment]
@MauriceArikoglu Thanks for notifiying me, I'm working on it

**Comment by MauriceArikoglu** [comment]
@okikio thanks for the app. It's honestly great. Just a bummer I can't use it on iPhone while on the go.. 👍 if you need help let me know, I'm down for having a look or call etc.

**Comment by okikio** [comment]
Yeah, I'm having trouble debugging this. Please try the beta version https://bundlejs.netlify.app does this have the same error from your side

**Comment by okikio** [comment]
@MauriceArikoglu After a bit of debugging it seems this issue only occurs on iOS 16 but on iOS 16.1 everything seems fine

**Comment by okikio** [comment]
I think this is resolved

**Comment by valtism** [comment]
Confirming this works on Safari on the beta version, but it does not work on the current prod version. Tried with private browsing too, and unregistering service workers.

**Comment by okikio** [comment]
Interesting...ok I'll try to backport a fix

**Comment by valtism** [comment]
Also, should mention this is Safari 16.3 on macOS.

**Comment by valtism** [comment]
Although testing on iOS also doesn't work. I think it works on beta because the popover doesn't close

**Comment by pketh** [comment]
just ran into this issue on macOS safari 17.2
